examples/ipynb/webgl_example_1.ipynb

```diff
@@ -16,11 +16,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
can you revert all changes in this file? it doesn't look like these changes are of any use